checklists/alz_checklist.en.json

```diff
@@ -2252,7 +2252,7 @@
             "service": "Key Vault",
             "guid": "5017f154-e3ab-4369-9829-e7e316183687",
             "id": "G02.01",
-            "severity": "High",
+            "severity": "Medium",
```

Comment:
I feel we can keep this one as high due to the risk associated with not storing secrets and keys in key vault